warp/builtins.py

```
@@ -4819,20 +4822,35 @@ def compute():
     :param start: The start of the ray in BVH space
-    :param dir: The direction of the ray in BVH space""",
+    :param dir: The direction of the ray in BVH space
+    :param root: The root to begin the query from
+    :param max_dist: The maximum distance along the ray to check for intersections""",
     export=False,
 )
 
 add_builtin(
     "bvh_query_next",
-    input_types={"query": BvhQuery, "index": int},
+    input_types={"query": BvhQuery, "index": int, "max_dist": float},
```

Ruff (0.13.1)

`BvhQuery` may be undefined, or defined from star imports

(F405)